### DISCUSSION TITLE
Базовая версия FHIR (R4 vs. R5) для профайлов

### DISCUSSION THREAD

**Discussion by shamilnn** [fhir-ru/core#36] [General]
Какую версию FHIR использовать как базовую - R4 или R5? 
В последней есть изменения, для которых в FHIR R4 приходилось использовать extensions.

**Comment by niquola** [comment]
@shamilnn а можешь самые существенные перечислить?

**Comment by shamilnn** [comment]
Это легче сделать если рассматривать конкретный ресур на [FHIR R5 Build](https://build.fhir.org/resourcelist.html) и внизу есть  _Compare to R4_. Например:
 - Practitioner.deceased[x] - новый элемент
 - Исчез MedicationStatement и появился MedicationUsage 
 - Immunization - вообще весь перекромсали
 
и т.д.

**Reply by ai-fhir** [reply]
Дорожная карта предполагает, что мы должны закончить кодирование ФРМО/ФРМР к маю 2022 года, публикация FHIR R5 ожидается во втором квартале 2022 года. Полагаю, что начать нам придется на R4. При этом ресурсы Organization, Location, Practitioner, PractitionerRole изменились слабо (биндинги к справочинкам и Practitioner.deceased[x]), поэтому не кажется, что в случае перехода на R5 мы столкнемся с большими сложностями. 

Общее предложение - обсудить переход на R5 после публикации R5. У нас был пример STU3, который сообщество практически проигнорировало, используя DSTU2, а затем сразу мигрировав на R4.
